src/components/PrimaryPanes/Outline.js

```diff
@@ -68,19 +68,18 @@ export class Outline extends Component<Props> {
     if (!selectedSource) {
       return;
     }
-
-    const sourceLine = func.location.start.line;
-    const functionText = getFunctionText(sourceLine);
-
     const copyFunctionItem = {
       id: "node-menu-copy-function",
       label: copyFunctionLabel,
       accesskey: copyFunctionKey,
-      disabled: !functionText,
+      disabled: false,
       click: () => {
+        const startLine = func.location.start.line;
+        const endLine = func.location.end.line;
```

Comment:
Destructuring would look nice here :)

Reply:
you think so?
```javascript
const {
          start: { line: startLine },
          end: { line: endLine }
        } = func.location;

Reply:
`EditorMenu.js` uses:

```js
const { location: { start, end } } = getFunctionLocation(sourceLine);
```

Reply:
Hahaha, ok, let's keep it this way only :)

Reply:
@darkwing why use `getFunctionLocation(sourceLine);` when we already have `func.location` ?